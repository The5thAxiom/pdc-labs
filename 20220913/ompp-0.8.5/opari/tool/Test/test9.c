int main() {
  int i;

#pragma \
 /*--*/ pomp \
 /*--*/ inst \
 /*--*/ init

#pragma pomp inst off

#pragma pomp inst begin(foo)

#pragma omp parallel
      i = 9;

#pragma pomp inst end(foo)

#pragma pomp inst on

#pragma pomp inst finalize

}
