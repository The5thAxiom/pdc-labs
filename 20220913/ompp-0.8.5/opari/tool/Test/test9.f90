      program test9

C$POMP  INST
C$POMP& INIT

      !$POMP INST OFF

      !$POMP INST &
      !$POMP BEGIN(foo)

      !$OMP PARALLEL
      i = 9
      !$OMP end PARALLEL

      !$POMP INST END(foo)

      !$POMP INST ON

   !$POMP INSTFINALIZE

      end
