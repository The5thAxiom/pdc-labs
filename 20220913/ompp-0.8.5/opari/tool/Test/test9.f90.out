#ifdef _POMP
#  undef _POMP
#endif
#define _POMP 200110

#line 1 "test9.f90"
      program test9

      call POMP_Init()
#line 5 "test9.f90"

      call POMP_Off()
#line 7 "test9.f90"

      call POMP_Begin(31)
#line 10 "test9.f90"

      call POMP_Parallel_fork(32)
#line 11 "test9.f90"
      !$omp parallel
      call POMP_Parallel_begin(32)
#line 12 "test9.f90"
      i = 9
      call POMP_Barrier_enter(32)
!$omp barrier
      call POMP_Barrier_exit(32)
      call POMP_Parallel_end(32)
#line 13 "test9.f90"
      !$omp end parallel
      call POMP_Parallel_join(32)
#line 14 "test9.f90"

      call POMP_End(31)
#line 16 "test9.f90"

      call POMP_On()
#line 18 "test9.f90"

      call POMP_Finalize()
#line 20 "test9.f90"

      end
